lldb/bindings/interface/SBAddressRangeExtensions.i

```diff
@@ -0,0 +1,11 @@
+%extend lldb::SBAddressRange {
+#ifdef SWIGPYTHON
+    %pythoncode%{
+      def __repr__(self):
+        import lldb
```

Comment:
Why does this import lldb? Shouldn't it already be imported?

Reply:
it's not, produces the following error:
`NameError: name 'lldb' is not defined`